easybuild/easyblocks/i/impi.py

```diff
@@ -58,6 +58,12 @@ def extra_options():
         }
         return IntelBase.extra_options(extra_vars)
 
+    def prepare_step(self):
+        if LooseVersion(self.version) >= LooseVersion('2017.2.174'):
+            super(EB_impi, self).prepare_step(requires_runtime_license=False)
```

Comment:
Doesn't matter, download of `impi` download for v2017.2.174 is identical, and doesn't allow you to provide a license file anymore either, so this change is required